reporting_app/static/datatable.js

```diff
@@ -92,6 +92,18 @@ var merge_multi_sources_keep_first = function(dt_config){
     return _merge_multi_sources(dt_config, merge_on_keep_first);
 }
 
+var test_exist = function(variable){
+    if ( variable instanceof Array ) {
+        return variable.length > 0;
+    }
+    return variable !== undefined && variable !== null && variable;
+}
+
+var color_filter = function( row, data, dataIndex ) {
+    if ( test_exist(data["trim_r1"]) || test_exist(data["trim_r2"]) || test_exist(data["tiles_filtered"]) ) {
```

Comment:
We could...
They're not necessarily columns though. They just need to be present in the json that the ajax call sent back.
In some case they're not being displayed as column.
I could have a list of column id provided by `dt_config` but that would be another variable to pass and I don't have a use-case for passing a different list at the moment.